apps/pagy_bare_rails.rb

```diff
@@ -68,18 +71,72 @@ class TestApp < Rails::Application # :nodoc:
   routes.draw do
     get '/' => 'test#index', as: 'test'
   end
+
+  ## Optional: MeiliSeach example
```

Comment:
```suggestion
  ## Optional: Meiliseach example
```

Reply:
agreed: Meilisearch - with sea**R**ch vs. seach - i'll make this change unless otherwise noted.

Reply:
Oh my, I didn't notice that! good catch! :p